[debug] [2019-07-09T22:06:58.385Z] ----------------------------------------------------------------------
[debug] [2019-07-09T22:06:58.389Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Javiera\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2019-07-09T22:06:58.390Z] CLI Version:   7.0.2
[debug] [2019-07-09T22:06:58.390Z] Platform:      win32
[debug] [2019-07-09T22:06:58.390Z] Node Version:  v10.15.3
[debug] [2019-07-09T22:06:58.392Z] Time:          Tue Jul 09 2019 18:06:58 GMT-0400 (GMT-04:00)
[debug] [2019-07-09T22:06:58.392Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=95837035&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-09T22:07:05.105Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-07-09T22:09:49.186Z] ----------------------------------------------------------------------
[debug] [2019-07-09T22:09:49.198Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Javiera\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2019-07-09T22:09:49.200Z] CLI Version:   7.0.2
[debug] [2019-07-09T22:09:49.200Z] Platform:      win32
[debug] [2019-07-09T22:09:49.200Z] Node Version:  v10.15.3
[debug] [2019-07-09T22:09:49.203Z] Time:          Tue Jul 09 2019 18:09:49 GMT-0400 (GMT-04:00)
[debug] [2019-07-09T22:09:49.203Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=781741161&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-09T22:09:57.909Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-07-10T18:13:54.465Z] ----------------------------------------------------------------------
[debug] [2019-07-10T18:13:54.468Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Javiera\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login --interactive
[debug] [2019-07-10T18:13:54.469Z] CLI Version:   7.0.2
[debug] [2019-07-10T18:13:54.469Z] Platform:      win32
[debug] [2019-07-10T18:13:54.469Z] Node Version:  v10.15.3
[debug] [2019-07-10T18:13:54.471Z] Time:          Wed Jul 10 2019 14:13:54 GMT-0400 (GMT-04:00)
[debug] [2019-07-10T18:13:54.471Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=140707527&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-10T18:13:59.819Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-07-10T18:14:51.917Z] ----------------------------------------------------------------------
[debug] [2019-07-10T18:14:51.921Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Javiera\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login:ci --interactive
[debug] [2019-07-10T18:14:51.921Z] CLI Version:   7.0.2
[debug] [2019-07-10T18:14:51.921Z] Platform:      win32
[debug] [2019-07-10T18:14:51.921Z] Node Version:  v10.15.3
[debug] [2019-07-10T18:14:51.923Z] Time:          Wed Jul 10 2019 14:14:51 GMT-0400 (GMT-04:00)
[debug] [2019-07-10T18:14:51.923Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=997514336&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-10T18:14:51.978Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:757:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-07-10T18:18:15.534Z] ----------------------------------------------------------------------
[debug] [2019-07-10T18:18:15.540Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Javiera\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login --debug
[debug] [2019-07-10T18:18:15.542Z] CLI Version:   7.0.2
[debug] [2019-07-10T18:18:15.542Z] Platform:      win32
[debug] [2019-07-10T18:18:15.542Z] Node Version:  v10.15.3
[debug] [2019-07-10T18:18:15.545Z] Time:          Wed Jul 10 2019 14:18:15 GMT-0400 (GMT-04:00)
[debug] [2019-07-10T18:18:15.545Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=353884264&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-10T18:18:28.825Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
